src/clusterfuzz/prune/__main__.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Reproduction module."""
+
+import argparse
+import os
+import shlex
+
+import clusterfuzz.fuzz
+
+
+def main():
+  os.environ['CF_INTERACTIVE'] = 'True'
+  os.environ['PROJECT_NAME'] = 'libClusterFuzz'
+
+  parser = argparse.ArgumentParser(description='Fuzzing tool')
+  parser.add_argument('-t', '--target', help='Path to target.', required=True)
+  parser.add_argument(
+      '-e',
+      '--engine',
+      help='Fuzzing engine.',
+      choices=clusterfuzz.fuzz.ENGINES,
+      default='libFuzzer')
+  parser.add_argument(
+      '-s',
+      '--sanitizer',
+      help='Sanitizer.',
+      choices=['address', 'memory', 'undefined'],
+      default='address')
+  parser.add_argument(
+      '-i',
+      '--input',
+      action='append',
+      help='Path to input corpus.',
+      required=True)
+  parser.add_argument(
+      '-o',
+      '--output',
+      help='Path to output (minimized) corpus.',
```

Comment:
I think we should try to be consistent and use the word "prune" or "minimize" but not both.

Reply:
good point, fixed.